amazon-lex-connector-product/pom.xml

```diff
@@ -26,6 +26,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
```

Comment:
one minor thing: i'd ad an '<id>include.openapi</id>' or something the like.
Makes the build more expressive. Quite helpful if you have to browse through the build logs and analyze a problem.